jsk_pcl_ros/cfg/OrganizedMultiPlaneSegmentation.cfg

```diff
@@ -8,7 +8,7 @@ from dynamic_reconfigure.parameter_generator_catkin import *;
 from math import pi
 
 gen = ParameterGenerator ()
-gen.add("min_size", int_t, 0, "the minimum number of the points of each cluster", 2000, 0, 10000)
+gen.add("min_size", int_t, 0, "the minimum number of the points of each cluster", 2000, 0, 2147483647)
```

Comment:
how did yuo caculate 2147483647 ? that is ~ (* 15731 15731). larger than 640 x 480, so what is the definition of 'recent high resolution cameras.'?

Suggested change is to add more commen. Something like 
"the minimum number of the points of each cluster (for high-reso cameras i.e. 1280x740, expected number of each cluster is 10% of total point ....." )

Reply:
this numeber is the max limit of int_t. i se this value because, for the firther future, we dont need to change this parameter any more because it is the limit.

Reply:
however we can change the value to 4K or 8K for the convenience.

Reply:
this PR only changes the max value of `min_size` and not changing the default value.

Reply:
sorry, I thought you changed the default value.. Sorry for that.